content/blog/posts/Summer-2016-Internship.rst

```diff
@@ -0,0 +1,94 @@
+OSL Summer 2016 Internship
+==========================
+:date: 2016-10-31
+:slug: summer-2016-internship
+:author: Amanda Kelner
+:img: Interns2016-1.jpg
+:tag: featured-stories
+
+In a new program, the Oregon State University Open Source Lab (OSL) partnered
+with Intel to give community college students an opportunity to get hands-on
+experience with open-source projects. This summer, four students interned for 10
+weeks at the OSL, working on a variety of projects, including web development
+and systems engineering, to advance their skills and knowledge with open-source
+work and communities.
+
+“This internship really has been a life-changing experience for me. I think it’s
+important to participate in more than just academics,” said Levi Willmeth from
+Linn-Benton Community College.
+
+That was exactly what the program organizers were hoping students would get out
+of the internship. Additionally, the project sought to increase diversity in the
+OSL and Oregon State’s computer science programs.
+
+“We decided to target the community colleges as they are historically more
+diverse, but largely underrepresented in these kinds of internships,” said Lance
+Albertson, the director of the Open Source Lab.
+
+The internships offered the students experiences beyond what they could have at
+their home schools. Intern Hannah Solorzano from Chemeketa Community College had
+never worked with open source. In fact, she did not know what it was before
+the internship. Although her interest was in cybersecurity, her professor
+encouraged her to apply.
+
+“The experience was very beneficial. I got to experiment with different
+languages, different platforms, and learned a lot of important tools,” Hannah
+said.
+
+The interns also appreciated the experience with a collaborative workflow.
+Working in groups is not encouraged in their school courses, so the internship
+gave them a sneak peak at the practical execution of real-world projects and
+access to collective support and assistance from their coworkers.
+
+This, however, created new challenges for the interns, including Levi who had to
+learn to adapt to a group of people working on the same section of code and
+adjust to integrated changes that conflicted with his own work. “Those little
+moments were where I learned a lot just from working with a team that I would
+never pick up on a school assignment,” he said.
+
+The interns spent half their time working on the systems engineering team and
+the other half on developer team. The systems projects included internal coding
+projects for which the interns learned a variety of tools including Git, and new
+coding languages like Ruby, and Chef. The developers project focused on redoing
```

Comment:
It's probably just a semantics thing, but I believe Chef is considered a tool instead of a language.  If you're looking for another language to mention, we also spent a couple weeks using Python.